README.md

```diff
@@ -1 +1,62 @@
-# Ark::Sponge
+<h1 align="center">Cryptographic Sponges</h1>
+
+<p align="center">
+    <a href="https://github.com/arkworks-rs/sponge/blob/master/LICENSE-APACHE">
+        <img src="https://img.shields.io/badge/license-APACHE-blue.svg"></a>
+    <a href="https://github.com/arkworks-rs/sponge/blob/master/LICENSE-MIT">
+        <img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
+</p>
+
+`ark-sponge` is a Rust library that provides infrastructure for implementing 
+*cryptographic sponges*. This library is released under the MIT License
+and the Apache v2 License (see [License](#license)).
+
+**WARNING:** This is an academic prototype, and in particular has not received careful code review.
+This implementation is NOT ready for production use.
+
+## Overview
+
+A cryptographic sponge is a cryptographic primitive that has two basic operations, *absorb* and *squeeze*. Sponges
+accept byte or field element inputs through its absorb operation. At any time, sponges may output bytes or field
+elements through its squeeze operation. Squeezed outputs are dependent on previous inputs and previous outputs.
+
+The library offers infrastructure for building cryptographic sponges and using them with different types of inputs.
```

Comment:
```suggestion
A cryptographic sponge is a cryptographic primitive that has two basic operations, *absorb* and *squeeze*. A sponge
accepts byte or field element inputs through its "absorb" operation. At any time, a user can invoke the "squeeze" operation on a sponge to obtain byte or field
element outputs. The sponge is stateful, so that squeezed outputs are dependent on previous inputs and previous outputs.

The library offers infrastructure for building cryptographic sponges and using them with different types of inputs.
```

Reply:
fixed